spark-doris-connector/spark-doris-connector-base/src/main/java/org/apache/doris/spark/client/write/AbstractStreamLoadProcessor.java

```diff
@@ -143,14 +144,26 @@ public AbstractStreamLoadProcessor(DorisConfig config) throws Exception {
     public void load(R row) throws Exception {
         if (createNewBatch) {
             if (autoRedirect) {
-                requestFuture = frontend.requestFrontends((frontEnd, httpClient) ->
-                        buildReqAndExec(frontEnd.getHost(), frontEnd.getHttpPort(), httpClient));
+                requestFuture = frontend.requestFrontends((frontEnd, httpClient) -> {
+                    if (isTwoPhaseCommitEnabled && frontEnd.getHttpPort() <= 0) {
+                        throw new IllegalArgumentException("option [" + DorisOptions.DORIS_FENODES.getName()
+                                + "] is not in correct format when ["
+                                + DorisOptions.DORIS_SINK_ENABLE_2PC.getName() + " = true"
+                                + "], for example: host:port[,host2:port]");
+                    }
```

Comment:
Why do we need to check isTwoPhaseCommitEnabled one more time here?

Reply:
It has been removed because of the nginx issue we discussed on WeChat, I mistakenly thought that the streamload node and the commit node must be the same